red4ext/src/conv.rs

```diff
@@ -219,7 +219,15 @@ where
     A::Repr: Default,
 {
     let mut init = A::Repr::default();
-    unsafe { ffi::get_parameter(frame, std::mem::transmute(&mut init)) };
+    unsafe {
+        ffi::get_parameter(
+            frame,
+            std::mem::transmute::<
+                &mut <A as crate::conv::FromRepr>::Repr,
+                red4ext_sys::interop::VoidPtr,
```

Comment:
add an import if it's missing
```suggestion
                VoidPtr,
```